samples/api/swapchain_recreation/swapchain_recreation.cpp

```diff
@@ -900,6 +900,7 @@ SwapchainRecreation::SwapchainRecreation()
 		add_instance_extension(VK_KHR_GET_SURFACE_CAPABILITIES_2_EXTENSION_NAME, true);
 		add_instance_extension(VK_EXT_SURFACE_MAINTENANCE_1_EXTENSION_NAME, true);
 		add_device_extension(VK_EXT_SWAPCHAIN_MAINTENANCE_1_EXTENSION_NAME, true);
+		has_maintenance1 = true;
```

Comment:
It was nicer to include a use_maintenance1, but the comments around has_maintenance1 attribute already covers the same intent